EvidenceApi/V1/UseCase/CreateDocumentSubmissionUseCase.cs

```diff
@@ -9,38 +9,26 @@
 using EvidenceApi.V1.Factories;
 using System.Threading.Tasks;
 using EvidenceApi.V1.Domain.Enums;
-using Notify.Exceptions;
-using Microsoft.Extensions.Logging;
-using System.Text.RegularExpressions;
 
 namespace EvidenceApi.V1.UseCase
 {
     public class CreateDocumentSubmissionUseCase : ICreateDocumentSubmissionUseCase
     {
         private readonly IEvidenceGateway _evidenceGateway;
-        private readonly IResidentsGateway _residentsGateway;
```

Comment:
This Use Case and tests were amended because `CreateDocumentSubmissionUseCase` no longer sends an email to the officer -- it's been moved to `SendNotificationUploadConfirmationToResidentAndStaff` (so doesn't need to have the other gateways injected)